src/lib/validateCurrentSession.ts

```diff
@@ -0,0 +1,33 @@
+import { cookies } from 'next/headers';
+const hankoApiUrl = process.env.NEXT_PUBLIC_HANKO_API_URL || '';
+
+export async function validateCurrentSession(){
+    const allCookies = await cookies();
+    const cookieToken = allCookies.get("hanko")?.value;
+
+    const validationOptions = { 
+        method: 'GET',
+        headers: {
+            'Cookie': `hanko=${cookieToken}` // If using cookie
+            // 'Authorization': `Bearer ${token}` // If using Authorization header
+        }
+    }
+
+      try {
+        const response = await fetch(hankoApiUrl + '/sessions/validate', validationOptions);
+    
+        if (!response.ok) throw new Error('Session validation failed');
+        
+        
+        const verifiedResponse = await response.json();
+        console.log(verifiedResponse)
+    
+        if(!verifiedResponse.is_valid) throw new Error('JWT is not valid');
+
+        return true;
+        
```

Comment:
Just return the result of `is_valid`. There is no need to throw an error.
```suggestion
        return verifiedResponse.is_valid;
```